account-service/build.gradle

```diff
@@ -7,7 +7,7 @@
 
 plugins {
   id 'java'
-  id 'org.springframework.boot' version '3.2.3'
+  id 'org.springframework.boot' version '3.2.4'
```

Comment:
This will need to be 3.2.5, there was a new CVE reported which was resolved Apr 18